modules/mfa/src/Auth/Process/Mfa.php

```diff
@@ -539,7 +542,7 @@ public static function validateMfaSubmission(
      *
      * @param array $state
      */
-    public static function redirectToMfaSetup(array &$state): void
+    public static function redirectToMfaSetup(array $state): void
```

Comment:
Arrays are already pointers. You can change the contents if you pass them. :)

Reply:
Are you sure? I didn't find any reference that backs that up.

Reply:
The interesting thing is that [SimpleSamlPHP uses both ways](https://github.com/simplesamlphp/simplesamlphp/blob/ced50409a35cf05870335e7fa5c25afceeb00b3c/src/SimpleSAML/IdP.php#L355), with and without a pointer. (Linked one file as an example, they do it more than just there.)

Reply:
As long as the function call is the same as the inherited one, that's fine.